.deepsource.toml

```diff
@@ -0,0 +1,18 @@
+version = 1
+
+test_patterns = [
+    "*/tests/**",
+    "*/test_*.py"
+]
+
+exclude_patterns = [
+    "doc/**",
+    "ci/**"
+]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
Is this necessary for deepsource to give useful results? I guess we can give it a try if you want to keep on making contributions using it, but I don't want to add it to CI at this time and if it doesn't get much use we'll probably eventually remove it.

Reply:
Sure